### ISSUE TITLE
White Block Outlines at night/in dark

### ISSUE THREAD

**Issue by Desenrad** [X0nk/Bliss-Shader#177]
While using DH 2.1 and the latest dev build as of 5/11/2024. I found major block outlines at Night and in dark. I confirmed it was Bliss specific by checking other DH supported shaders. 
![image](https://github.com/X0nk/Bliss-Shader/assets/69370875/508a42e8-2a3e-4d61-91b6-d138c90e5aef)

**Comment by X0nk** [comment]
what in tarnation
does this happen on the latest in-development version of bliss?
does this happen without additional mods?

**Comment by Desenrad** [comment]
This is in a modpack I made. I'll set up a copy with only DH and Bliss to test.
Fabric v1.20.1
![image](https://github.com/X0nk/Bliss-Shader/assets/69370875/b376e6c1-b5bd-4064-b306-703b8dddb0f7)
Looks like it may have been some combination of mods I had. I tested with a shorter list and the bug was not present. Might continue to look into what mod caused the bug. Any ideas which type of mods I can narrow down?
![image](https://github.com/X0nk/Bliss-Shader/assets/69370875/4b2e8668-768a-4812-82ca-8227f0c8380a)
